Library/ViewModels/DashboardViewModel.swift

```diff
@@ -136,11 +191,11 @@ public final class DashboardViewModel: DashboardViewModelInputs, DashboardViewMo
     }
 
     let drawerStateProjectsAndSelectedProject = Signal.merge(
-      projectsAndSelected.map { ($0, $1, false) },
-      projectsAndSelected.takeWhen(self.showHideProjectsDrawerProperty.signal).map { ($0, $1, true) }
-      )
+      projectsAndSelected.map { ($0.0, $0.1, false) },
```

Comment:
Any reason why this is using tuple restructuring now? We generally try to avoid that as it's less readable.